Source/InstrumentEditorVRC7.cpp

```diff
@@ -30,9 +30,52 @@
 #include "InstrumentEditorVRC7.h"
 #include "Clipboard.h"
 
-static unsigned char default_inst[(16+3)*16] = 
+#include "FamiTracker.h"
+#include "Settings.h"
+
+// based off NSFPlay emu2413's hardware patch scheme
+#define OPLL_TONE_NUM 9
+static unsigned char default_inst[OPLL_TONE_NUM][(16 + 3) * 8] =
```

Comment:
- [ ] This should be `static const` unless you intend to mutate the contents of the array.
- [ ] if you make it a const, the name should be uppercased to make it clear it's not a local variable and is const. if not, maybe do it anyway? (rust capitalizes non-const globals, i'm not sure whether to do so or not in C++.)
- [ ] maybe the name `DEFAULT_PATCHES` will work better. i haven't tried.